e2e/tests/desktop/registered-shopper.spec.js

```diff
@@ -15,85 +23,12 @@ const {
 const REGISTERED_USER_CREDENTIALS = generateUserCredentials();
 
 test("Registered shopper can checkout items", async ({ page }) => {
```

Comment:
Can we start adding a comment block above each test explaining what the flow is ?
It might feel redundant now since the tests are relatively simple. But as the tests get more elaborate and complex we'll have to read through the entire code to figure out what is the test doing. Something like [this](https://github.com/SalesforceCommerceCloud/pwa-sfra-hybrid-tests/blob/6a0f1a4596dccc99bf1c7efbce8406a1da577d78/tests/mobile/auth.spec.js#L74-L76)

Reply:
Added documentation